src/pages/components/text-input/style.mdx

```diff
@@ -326,6 +327,44 @@ These sizes apply only to the default style of text input.
 
 <Caption>Text input default style sizes | px / rem</Caption>
 
+## AI Presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the default variant.
+
+[Learn more about the AI style elements]()
+
+| Element         | Property         | Token / Size      |
+| --------------- | ---------------- | ----------------- |
+| Linear gradient | start            | $ai-aura-start-sm |
+|                 | stop             | $ai-aura-stop     |
+| Field           | border-bottom    | $ai-border-strong |
+|                 | background color | $field\*          |
```

Comment:
The tokens should be in the code snippets. I didn't have this in my spec, my bad.
```suggestion
| Linear gradient | start            | `$ai-aura-start-sm` |
|                 | stop             | `$ai-aura-stop`     |
| Field           | border-bottom    | `$ai-border-strong` |
|                 | background color | `$field`\*          |
```